scripts/junit2issue.py

```diff
@@ -0,0 +1,206 @@
+    '''
+    tod = datetime.datetime.now()
+    d = datetime.timedelta(days=60)
+    a = tod - d
+    open_issues = repository.get_issues(state='open', since=a, labels=['bug'])
+    for _issue in open_issues:
+        if report['CASE_NAME'] in _issue.title:
+            return _issue
+        if report['LOGS'] and report['LOGS'] in _issue.body:
+            return _issue
+
+    return None
+
+def get_desc_template():
+    '''
+        get_desc_template
+    '''
+    m = imp.load_source('GITHUB_TEMPLETE', 'github_issue_template.tmpl')
+    return m.GITHUB_TEMPLETE
+
+def get_title_template():
+    '''
+        get_title_template
+    '''
+    m = imp.load_source('GITHUB_ISSUE_TITLE_TEMPLETE', 'github_issue_template.tmpl')
+    return m.GITHUB_ISSUE_TITLE_TEMPLETE
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.DEBUG,
+                        format='[%(filename)s:%(lineno)d] %(threadName)s %(message)s')
+
+    auto_report = False
+    parser = argparse.ArgumentParser(description='Process parameters')
+    parser.add_argument('-v', '--vfolder', dest='vfolder', help='version folder')
+    parser.add_argument('-i', '--pfile', dest='pfile', help='platform report file', required=True)
+    parser.add_argument('-a', '--auto_report', dest='auto_report', help='auto report')
+    parser.add_argument('-r', '--repo', dest='report_repo', help='report repo', required=True)
+
+    args = parser.parse_args()
+
+    logging.info(args)
+    pfile = args.pfile
+    vfolder = args.vfolder
+    report_repo = args.report_repo
+
+    xml_file = os.path.join('..\\results', vfolder, pfile)
+    _xml = JUnitXml.fromfile(xml_file)
+    #reports = parser_testsuites_for_result(_xml, Error)
+    reports = parser_testsuites_for_result(_xml, Failure)
+    logging.info(len(reports))
+    if not auto_report:
+        print("Do you want to report issue?Y/N")
+        _choose = input()
+        if _choose not in ['y', 'Y']:
+            print("Bye\n")
+            sys.exit(0)
+
+    g = github_login()
+    repo = g.get_repo(report_repo)
+    label1 = repo.get_label("area: Tests")
+    label2 = repo.get_label("bug")
+    _template = get_desc_template()
+    _title_tmpl = get_title_template()
+
+    for rep in reports:
+        if not auto_report:
+            print("\r\ndo you want to report ", rep['CASE_NAME'])
+            _choose = input()
+            if _choose in ['n', 'N']:
+                continue
+        issue = check_issue_existing(repo, rep)
+        if not issue:
+            issue_content = gen_content_from_template(_template, rep)
+            # logging.debug(issue_content)
+            issue_title = gen_content_from_template(_title_tmpl, rep)
+            repo.create_issue(title=issue_title, body=issue_content, labels=[label1, label2])
+        else:
+            logging.info(_title_tmpl)
+            logging.info(rep)
+            issue_title = gen_content_from_template(_title_tmpl, rep)
+            issue.create_comment(issue_title)
```

Comment:
The template of a title is not the most useful here. Something like:
`f"Also fails on {PLATFORM} for {ZEPHYR_VERSION}"` might be more useful

Reply:
good point, let me update